fossunited/chapters/doctype/conference_tickets/conference_tickets.json

```diff
@@ -173,7 +173,7 @@
    "print": 1,
    "read": 1,
    "report": 1,
-   "role": "System Manager",
+   "role": "All",
```

Comment:
System Manager Role needs to be there too, along with All, do not replace the prior with the latter.

Reply:
Have added both the "System Manager" and "All" roles.